Software/src/globals.h

```diff
@@ -76,7 +76,7 @@ namespace spartan {
         ERROR_DATAFORMAT,
         ERROR_READ,
         ERROR_POLL,
-        ERROR_INVALID_STATUS
+        ERROR_INVALID_STATUS,
```

Comment:
Since we no longer have the other error code you added before, can you remove this extra unneeded comma? I think there's another extra comma in the `flight_state` enum too:

https://github.com/Bruin-Spacecraft-Group/SPARTAN/blob/2286670ea354264dca37964ee24333aeba0529d8/Software/src/globals.h#L61-L68